File: java/AbstractFactory/complete/ModernDocumentFactory.java
class ModernDocumentFactory extends DocumentFactory{

  public Resume create_resume(String content){
    //returns a new LegalResume object with the content parameter
    return new ModernResume(content);
  }

  public Letter create_letter(String content){
    //returns a new LegalLetter object with the content parameter
    return new ModernLetter(content);
  }
}

File: python/AbstractFactory/complete/ModernDocumentFactory.py
from DocumentFactory import DocumentFactory
from ModernDocuments import *

class ModernDocumentFactory(DocumentFactory):

    def create_resume(self, content):
        '''
        returns a new ModernResume object with the content parameter
        '''
        return ModernResume(content)

    def create_letter(self, content):
        '''
        returns a new ModernLetter object with the content parameter
        '''
        return ModernLetter(content)
